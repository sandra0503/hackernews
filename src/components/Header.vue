<template>
  <header class="header">Hackernews Top 500</header>
</template>

<script>
export default {
  name: "Header"
};
</script>

<style lang="scss" scoped="true">
.header {
  background: #009688;
  color: #fff;
  font-size: 24px;
  font-weight: 600;
  padding: 20px;
}
</style>
