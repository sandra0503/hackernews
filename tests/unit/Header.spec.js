import { shallowMount } from "@vue/test-utils";
import Header from "@/components/Header";

describe("Header component", () => {
  const wrapper = shallowMount(Header);

  it("renders the component and matches snapshot", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
