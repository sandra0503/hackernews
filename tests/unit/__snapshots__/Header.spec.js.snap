// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders the component and matches snapshot 1`] = `<header class="header">Hackernews Top 500</header>`;
